packages/admin/docs/07-appearance.md

```diff
@@ -65,6 +65,23 @@ In `config/filament.php`, set the `layouts.sidebar.is_collapsible_on_desktop` to
 ],
 ```
 
+If you enable the collapsible sidebar, make sure to load the Tippy CSS-files as well, by adding the following to your service provider:
```

Comment:
If it's installed by default, why do we need this?

Reply:
We need this when people create their own theme.

Reply:
I meant the first clause about using `Filament::register`, before theming is mentioned?

Reply:
Oh sure. I think that whole part belongs in the "Building themes" section.